client/src/components/HolidayTile.js

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+
+const HolidayTile = (props) => {
+    
+    return (
+        <div>
+            <li> 
+                {props.name}: {props.date}
+            </li>
+        </div>
+    )
+} 
+
```

Comment:
At the moment, there isnt a need for a <div> AND an <li>. Choose one or the other!